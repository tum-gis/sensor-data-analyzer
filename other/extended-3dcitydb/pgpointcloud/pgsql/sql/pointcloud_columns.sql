INSERT INTO pointcloud_formats (pcid, srid) VALUES (777, 666);
CREATE TABLE pc1(p pcpoint);
CREATE TABLE pc2(p pcpoint);
DELETE FROM pointcloud_formats WHERE pcid = 777;
SELECT * from pointcloud_columns ORDER BY 1,2,3,4;
DROP TABLE pc1;
DROP TABLE pc2;
