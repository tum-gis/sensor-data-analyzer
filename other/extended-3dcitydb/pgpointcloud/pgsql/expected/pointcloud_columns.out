INSERT INTO pointcloud_formats (pcid, srid) VALUES (777, 666);
CREATE TABLE pc1(p pcpoint);
CREATE TABLE pc2(p pcpoint);
DELETE FROM pointcloud_formats WHERE pcid = 777;
SELECT * from pointcloud_columns ORDER BY 1,2,3,4;
 schema | table | column | pcid | srid |  type   
--------+-------+--------+------+------+---------
 public | pc1   | p      |      |      | pcpoint
 public | pc2   | p      |      |      | pcpoint
(2 rows)

DROP TABLE pc1;
DROP TABLE pc2;
